src/components/accordion/components/AccordionItem.vue

```diff
@@ -199,7 +199,7 @@ export default {
     border: 0;
 
     &.vs-accordion-item__responsive {
-        border-top: 1px solid $vs-color-border;
+        border-top: 1px solid $vs-color-new-border-primary;
```

Comment:
If the token is changing to include "primary" can't we omit "new" from the namespacing? Surely the only reason for having "new" is to differentiate it from an otherwise identical token?

Reply:
@markup-mitchell Hey, I said up in the description, the next PR will be to remove the old tokens and the "new" part. 

This was started when we were half using new and old tokens and we couldn't have "$vs-color-border-primary" as the old one still existed. So yeah, will work on updating this separately.

Reply:
ok sorry